libcxx/include/string

```diff
@@ -1748,24 +1750,11 @@ private:
 
     _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX20 void __default_init() {
         __r_.first() = __rep();
```

Comment:
I think we can just inline this now. No need to keep the function for this.

Reply:
Done.